composer.json

```diff
@@ -48,6 +49,10 @@
         "drupal": {
             "type": "composer",
             "url": "https://packages.drupal.org/8"
+        },
+        "oe_corporate_blocks": {
+            "url": "https://github.com/openeuropa/oe_corporate_blocks.git",
+            "type": "git"
```

Comment:
The module is not on Packagist: https://packagist.org/packages/openeuropa/oe_corporate_blocks

Reply:
Fixed.